ddp-workspace/projects/ddp-singular/src/app/app.component.ts

```diff
@@ -13,11 +15,16 @@ import { IGNORE_ANALYTICS_CLASS } from './constants/analytics';
 })
 export class AppComponent {
   title = 'ddp-singular';
+  isProdMode: boolean;
 
   constructor(
     private elRef: ElementRef,
-    private analytics: AnalyticsEventsService
-  ) {}
+    private analytics: AnalyticsEventsService,
+    private dialog: MatDialog,
+    @Inject('ddp.config') private config: ConfigurationService
+  ) {
+    this.isProdMode = this.config.logLevel.toString() === '2';
```

Comment:
Let's just set it in pepperConfig.js . 
runtimeEnvironment = 'prod' | 'dev' | 'staging' | 'test'.
And let's pick it up via ConfigurationService the way we pickup the Auth0Domain or anything else that comes from pepperConfig.js.
You will have to modify ddp-angular/config/pepperConfig.js.ctmpl . Should not be too hard.

Reply:
Done.